services/passport.js

```diff
@@ -10,34 +10,38 @@ passport.use(
     {
       clientID: process.env.GOOGLE_CLIENT_ID,
       clientSecret: process.env.GOOGLE_CLIENT_SECRET,
-      callbackURL: "/auth/google/callback",
+      callbackURL: "/user/login/google/callback",
```

Comment:
I guess /user/login/ is enough

Reply:
Actually I want to change this end point to be api/login/google/callback, because I want to distinguish between the frontend and backend points. Knowing that /google is important here because I want to add another authentication later by email and password and I may add a facebook authentication.

Reply:
I'm not sure why you want to distinguish between backend endpoints and frontend routing?
This is the server and I don't think the frontend has anything to do with it, anyway `/user/google-login` can be the endpoint

Reply:
This is because I'm using proxySetup file in the frontend, and when the end points of the front and back ends match, I think this will make some kind of confusion.

Reply:
I don't think so, it's right that those proxies in the frontend but they're pointing to the backend endpoint
Naming your backend enpoints should not be concerned with anything else espacially your frontend